api/address/routes.go

```diff
@@ -39,40 +40,88 @@ func Routes(router *wrapper.RouterWrapper) {
 func GetAccount(c *gin.Context) {
 	ef, ok := c.MustGet("elrondFacade").(FacadeHandler)
 	if !ok {
-		c.JSON(http.StatusInternalServerError, gin.H{"error": errors.ErrInvalidAppContext.Error()})
+		c.JSON(
+			http.StatusInternalServerError,
+			shared.GenericAPIResponse{
+				Data:  nil,
+				Error: errors.ErrInvalidAppContext.Error(),
+				Code:  string(shared.ReturnCodeInternalError),
```

Comment:
Also, return code suggests integer, so I would use integers for this. If we have the `ReturnCode` type, it could implement `Stringer` interface. We can have a method: `FormatError(code ReturnCode, err error)` that returns `code + ":" + err.Error()`, and this can be our Error for `GenericAPIResponse`.

Reply:
If I would set it to `ReturnCode` type, why would this help us? You can see the return code of each API call without the need to be included in the response payload. The idea of return code is that we can define custom return codes that can be customized in the future. For now, I only 'copied' the 200, 400 and 500 return codes. 
@andreibancioiu can you please provide an input?

Reply:
I was saying to use the ReturnCode type directly in the structure so you don’t have to do string(ReturnCode)

Reply:
Done. also, changed return codes messages to snake case